components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -178,23 +177,22 @@ open class GleanInternalAPI {
      * https://developer.android.com/reference/android/view/accessibility/AccessibilityManager.html
      * @param accessibilityManager The system's [AccessibilityManager] as
      * returned from applicationContext.getSystemService
-     * @param packageManager The system's [PackageManager], as from
-     * applicationContext.packageManager
-     * @returns services A list of strings describing the enabled services. If
+     * @returns services A list of ids of the enabled accessibility services. If
      *     the accessibility manager is disabled, returns null.
      */
     internal fun getEnabledAccessibilityServices(
-        accessibilityManager: AccessibilityManager,
-        packageManager: PackageManager
+        accessibilityManager: AccessibilityManager
     ): List<String>? {
         if (!accessibilityManager.isEnabled) {
             logger.info("AccessibilityManager is disabled")
             return null
         }
         return accessibilityManager.getEnabledAccessibilityServiceList(
             AccessibilityServiceInfo.FEEDBACK_ALL_MASK
-        ).map {
-            it.loadDescription(packageManager)
+        ).mapNotNull {
+            // Note that any reference in java code can be null, so we'd better
+            // check for null values here as well.
+            it.id
```

Comment:
Looking at the code of `AccessibilityServiceInfo` it looks like the id can never be null. But it's probably safer to keep the check.

Reply:
I see -- so we're using `id`s rather than descriptions, since descriptions are localized.  Makes sense.